packages/xo-web/src/xo-app/backup-ng/restore/metadata.js

```diff
@@ -0,0 +1,274 @@
+    strongConfirm: {
+      messageId: 'bulkDeleteMetadataBackupsConfirmText',
+      values: {
+        nMetadataBackups: reduce(
+          entries,
+          (sum, entry) => sum + entry.backups.length,
+          0
+        ),
+      },
+    },
+  }).then(
+    () => deleteMetadataBackups(flatMap(entries, ({ backups }) => backups)),
+    noop
+  )
+}
+
+// ---------------------------------------------------------------------------
+
+const ACTIONS = [
+  {
+    handler: bulkRestore,
+    icon: 'restore',
+    individualHandler: restore,
+    label: _('restore'),
+    level: 'primary',
+  },
+  {
+    handler: bulkDelete,
+    icon: 'delete',
+    individualHandler: delete_,
+    label: _('delete'),
+    level: 'danger',
+  },
+]
+
+const COLUMNS = [
+  {
+    name: _('type'),
+    valuePath: 'type',
+  },
+  {
+    name: _('item'),
+    valuePath: 'id',
+  },
+  {
+    name: _('firstBackupColumn'),
+    itemRenderer: ({ first }) => (
+      <FormattedDate
+        value={new Date(first.timestamp)}
+        month='long'
+        day='numeric'
+        year='numeric'
+        hour='2-digit'
+        minute='2-digit'
+        second='2-digit'
+      />
+    ),
+    sortCriteria: 'first',
+    sortOrder: 'desc',
+  },
+  {
+    name: _('lastBackupColumn'),
+    itemRenderer: ({ last }) => (
+      <FormattedDate
+        value={new Date(last.timestamp)}
+        month='long'
+        day='numeric'
+        year='numeric'
+        hour='2-digit'
+        minute='2-digit'
+        second='2-digit'
+      />
+    ),
+    sortCriteria: 'last',
+    default: true,
+    sortOrder: 'desc',
+  },
+  {
+    name: _('availableBackupsColumn'),
+    valuePath: 'available',
+  },
+]
+
+export default decorate([
+  addSubscriptions({
+    remotes: cb =>
+      subscribeRemotes(remotes => {
+        cb(toArray(remotes))
+      }),
+  }),
+  provideState({
+    computed: {
+      // {
+      //   [JobName | poolId]: {
+      //     available: Number,
+      //     backups: Array,
+      //     first: Number,
+      //     id: Node | String,
+      //     label: String,
+      //     last: Number,
+      //     type: 'XO' | 'pool',
+      //   }
+      // }
+      async backups(_, { remotes = [] }) {
+        if (remotes.length === 0) {
+          return
+        }
+
+        const { xo, pool } = await listMetadataBackups(remotes)
+
+        const collection = {}
+        forOwn(xo, entries =>
+          entries.forEach(entry => {
+            const jobName = entry.jobName
+            let backup = collection[jobName]
+            if (backup === undefined) {
+              backup = collection[jobName] = {
+                backups: [],
+                id: `Xen Orchestra(${jobName})`,
+                label: jobName,
+                type: 'XO',
+              }
+            }
+
+            backup.backups.push(entry)
+          })
+        )
+        forOwn(pool, entriesByRemote =>
+          forOwn(entriesByRemote, (entriesByPool, poolId) => {
```

Comment:
The naming is weird. How can you get `poolId` as a key in a collection called `entriesByRemote`?